RockPaperScissors/views/index.erb

```diff
@@ -0,0 +1,25 @@
+<h1>GO!</h1>
+<h2>Put in your name!</h2>
+<form action="/result" method="post">
+    <label for="player_1_name"> </label> <br>
+    Player 1
```

Comment:
this would work with what I suggested above
```
<input type='submit' name='weapon' value='rock'>
<input type='submit' name='weapon' value='paper'>
<input type='submit' name='weapon' value='scissors'>
```